spec/lucky/input_helpers_spec.cr

```diff
@@ -224,6 +233,26 @@ describe Lucky::InputHelpers do
     <textarea id="user_first_name" name="user:first_name" rows="5" cols="15">My name</textarea>
     HTML
   end
+
+  it "renders time inputs" do
+    view.time_input(form.joined_at).to_s.should contain <<-HTML
+    <input type="time" id="user_joined_at" name="user:joined_at" value="">
+    HTML
+
+    view.time_input(form.joined_at, min: "09:00", max: "18:00").to_s.should contain <<-HTML
+    <input type="time" id="user_joined_at" name="user:joined_at" value="" min="09:00" max="18:00">
+    HTML
+  end
+
+  it "renders date inputs" do
+    view.date_input(form.joined_at).to_s.should contain <<-HTML
+    <input type="date" id="user_joined_at" name="user:joined_at" value="">
+    HTML
+
+    view.date_input(form.joined_at, min: "2019-01-01", max: "2019-12-31").to_s.should contain <<-HTML
+    <input type="date" id="user_joined_at" name="user:joined_at" value="" min="2019-01-01" max="2019-12-31">
+    HTML
```

Comment:
I think this should also have a spec for when a value is on the attribute. So we know that the HTML value is set correctly. Right now. This may mean implementing a custom version of the method but I’m not sure